docs/commands.rst

```diff
@@ -291,6 +291,18 @@ negation with some priority level.
 
    notation ¬ prefix 5;
 
+*Remarks:*
+
+* Prefix and infix operators share the same levels of priority, hence depending
+  on the binding power, ``-x + z`` may be parsed ``(-x) + z`` or ``-(x + z)``.
+
+* Regular application has a higher binding power than operators application:
+  ``- f x`` is always parsed ``- (f x)``, whatever binding power is set for
```

Comment:
prefix operator application

Reply:
Prefix and infix, in fact, we have `f x + g y` = `(f x) + (g y)` as well.